feat: LEAP-598: Self Serve M2

robot-ci-heartex opened the pull request:
Implement limitations on data import for Expired Starter Cloud

juliosgarbi committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8085785259

juliosgarbi committed:
Merge branch 'develop' into 'fb-leap-598'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8098316925

juliosgarbi committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8098316925

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8098347239

juliosgarbi commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8441286285)
> Successfully merged:  147 files changed, 5437 insertions(+), 1427 deletions(-)

juliosgarbi committed:
Merge branch 'develop' into 'fb-leap-598'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8441286285

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8441306627

juliosgarbi commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8445187019)
> Successfully merged: Already up to date.

juliosgarbi commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8451581637)
> Successfully merged:  6 files changed, 16 insertions(+), 8 deletions(-)

juliosgarbi committed:
Merge branch 'develop' into 'fb-leap-598'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8451581637

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/8451608102
